src/sentry/templates/sentry/emails/activity/release_summary.html

```diff
@@ -0,0 +1,37 @@
+{% extends "sentry/emails/activity/generic.html" %}
+
+{% load sentry_avatars %}
+{% load sentry_helpers %}
+
+{% block action %}
+{% endblock %}
+
+{% block activity %}
+    <h2 style="margin-bottom: 10px">
+        Release {{ version_parsed }} has been deployed to {{ environment }} for an hour
+    </h2>
+    <p class="muted"><small><strong style="color: #72697d;">{{ deploy.date_finished }}</strong> <span class="divider">&nbsp;</span>  {{ commit_count }} commit{{ commit_count|pluralize }}, {{ author_count }} author{{ author_count|pluralize }}, and {{ file_count }} file{{ file_count|pluralize }} changed across {{ project_count }} project{{ project_count|pluralize }}</small></p>
+
+    <hr>
+
+    <h6>New Issues From This Release</h6>
```

Comment:
This text appears above a list of projects which I think can be a bit confusing to a reader, since the button to actually take them to the issues is not near this. Probably just me being nitpicky though, it's not a blocker.